sdks/java/io/hdfs/src/main/java/org/apache/beam/sdk/io/hdfs/HDFSFileSink.java

```diff
@@ -265,6 +265,11 @@ public String close() throws Exception {
     }
 
     @Override
+    public void abort() throws Exception {
+      close();
```

Comment:
I don't familiar with this code, and looks to me there is no tests. So, I prefer not to change it in this PR.

(I made the change deleting the file here, but stashed it.)